libtraceevent/event-parse.c

```diff
@@ -2905,6 +2905,7 @@ process_arg_token(struct event_format *event, struct print_arg *arg,
 		break;
 
 	case EVENT_DQUOTE:
+		/* fall through */
 	case EVENT_SQUOTE:
```

Comment:
Hmm.. I think this is normal for a switch case.  Does GCC7 shows warning for this too?